tests/test_all.py

```diff
@@ -238,6 +239,12 @@ def test_zypper_dup_works(container_per_test: ContainerData) -> None:
         "skelcd-EULA-bci",
         "sles-release",
         "ALP-dummy-release",
+        # packages pulled into the kernel-module container:
+        "kernel-default-devel",
```

Comment:
lets update the BCI repo instead and then we can remove the allowlisting here.

Reply:
I have removed it as Riccardo & Fred have added these packages into the repo.